Logos/contrastsecurity_logo.svg

```diff
@@ -0,0 +1 @@
-.2.7-2.9 8.7-3.2 9.4h6.3l-3.1-9.4zm14.8 20.6c-.1.2-.4.2-.8.2h-5.9c-.3 0-.6-.1-.7-.2-.2-.1-.3-.3-.4-.6-.3-.7-1.3-3.5-1.6-4.1h-10.9c-.2.7-1.3 3.5-1.6 4.1-.2.5-.5.8-1.1.8h-6.1c-.2 0-.5-.1-.6-.2-.2-.2-.2-.3-.1-.6L159 14.9c0-.1.1-.2.1-.3l.3-.3c.1-.1.2-.1.3-.2.1-.1.2-.1.3-.1h5.3c.2 0 .4.1.6.2.2.1.3.3.3.4l.5 1.2 10.8 27.8c.2.3.2.6.1.7m-40.1-20.1c0 .4-.1.9-.2 1.2-.1.4-.3.8-.6 1.1-.2.3-.6.6-1 .9s-.8.3-1.3.4c-.2.1-.5.1-.6.1s-.4.1-.7.1h-3.8v-7.5h2.8c.4 0 .8 0 1.1.1.3.1.7.1 1.1.1.5.1.9.2 1.2.5.4.2.7.5 1 .9.3.3.5.7.6 1.1.3.2.4.6.4 1m9 19.7c0-.2-.1-.4-.2-.7-.1-.2-.3-.5-.3-.7l-5.3-10c.7-.4 1.4-.9 2-1.5.6-.6 1.2-1.2 1.6-2 .5-.7.8-1.5 1.1-2.4.2-.9.4-1.8.4-2.7 0-1.4-.2-2.7-.8-3.9-.5-1.2-1.2-2.3-2.2-3.2-.9-.9-2-1.6-3.2-2.2-1.2-.5-2.6-.8-4-.8h-13.3c-.3 0-.5.1-.7.2-.1.2-.2.4-.2.7v28.7c0 .3.1.5.2.7.2.1.4.2.7.2h6.1c.6 0 1-.3 1-.9v-9.2h3.3l5 9.4.3.6c.2.1.4.2.6.2h7.1c.2 0 .5-.1.6-.2.1.1.2-.1.2-.3M93.2 15c0-.3.1-.5.3-.7.2-.2.4-.2.7-.2h22.1c.2 0 .5.1.7.2.2.1.3.4.3.7v5.1c0 .2-.1.5-.2.7-.2.2-.4.3-.7.3h-7.1v22.5c0 .2-.1.5-.3.7-.2.2-.4.3-.7.3h-6c-.2 0-.5-.1-.7-.3-.2-.2-.3-.4-.3-.7V21.1h-7.1c-.3 0-.5-.1-.7-.3s-.3-.4-.3-.8v-5zM62.1 43.7c0 .2.1.5.2.6.1.2.4.2.7.2h5.8c.3 0 .6-.1.7-.2.1-.1.2-.4.2-.7V26.1c1.8 3 9 14.9 10.8 17.9.1.2.3.3.4.5.1.1.3.2.5.2H88c.3 0 .5-.1.7-.2.2-.1.2-.3.2-.6V15c0-.3-.1-.5-.3-.7-.2-.1-.4-.2-.7-.2h-5.8c-.2 0-.4.1-.6.2-.2.1-.3.3-.3.6v16.4c-1.8-2.8-8.6-13.8-10.4-16.6-.1-.2-.3-.4-.5-.5-.2-.1-.4-.1-.6-.1H63c-.3 0-.5.1-.7.3-.1.2-.2.4-.2.6v28.7zM41 38.6c0-.4.1-1.8 1.5-1.8 0 0 4.3-.2 6.3-3.2 2.1-3.1 1.2-7.1-.5-9.3-2.1-2.7-5.2-2.6-5.4-2.6-.2 0-1.9.1-1.9-1.5v-4.8c0-.3.1-1.6 1.6-1.6 4.5 0 7.7 1.4 10.4 3.5 3.9 3.1 5.7 8.1 5.7 11.6 0 3.3-.6 4.7-1.1 6.3-.7 2.2-2.8 4.8-4.1 6-5.1 4.3-10.8 3.8-10.8 3.8s-1.5-.1-1.5-1.6c-.2-1-.2-4.8-.2-4.8z"/><path class="st1" d="M37 3.4v51h-1.9v-51zM217.3 63.2l4.8-8.7h-2.3l-3.7 6.4-3.7-6.4h-2.2l4.9 8.8v4.3h2.2v-4.4zM203 56.5v-2h-10.5v2h4.1v11.1h2.2V56.5h4.2zm-19.4-2h-2.2v13.1h2.2V54.5zM163 61.6v-5.2h3.1c2 0 3.2.8 3.2 2.6s-1.2 2.7-3.2 2.7l-3.1-.1zm6.2 6h2.5l-2.9-4.4c1.8-.7 2.7-2.2 2.7-4.3 0-2.8-1.9-4.5-5.3-4.5h-5.3v13.1h2.2v-4h3.1c.2 0 .4 0 .6-.1l2.4 4.2zm-18.5-13.1h-2.2v7.7c0 2.2-1.4 3.5-3.5 3.5-2.2 0-3.5-1.4-3.5-3.5v-7.7h-2.2v7.7c0 3.4 2.2 5.5 5.7 5.5s5.7-2.1 5.7-5.5v-7.7zm-25.5-.1c-3.9 0-6.9 2.8-6.9 6.6s2.9 6.7 6.8 6.7c1.8 0 3.7-.9 5.1-2.2l-1.3-1.5c-1 1-2.3 1.5-3.6 1.5-2.6 0-4.7-2-4.7-4.6 0-2.6 2-4.6 4.7-4.6 1.4 0 2.7.6 3.6 1.6l1.3-1.6c-1.2-1.1-3-1.9-5-1.9m-15.7 2.1v-2h-9.4v13.1h9.6v-2h-7.4V62h6.4v-2h-6.4v-3.5h7.2zm-23.7-2.1C83 54.4 81 55.8 81 58c0 4.6 7.1 3.1 7.1 6 0 1-.9 1.6-2.4 1.6-1.4 0-3.1-.7-4.4-1.9l-1 1.9c1.4 1.2 3.3 2.1 5.4 2.1 2.9 0 4.9-1.5 4.9-3.8.1-4.7-7.1-3.2-7.1-6 0-.9.9-1.4 2.1-1.4 1 0 2.4.3 3.9 1.2l.9-1.9c-1.2-.9-2.9-1.4-4.6-1.4"/><path d="M31 38.4s-.1-1.5-1.5-1.5h-7c-3 0-5-.2-7-3.2-2-3.1-1.2-7 .5-9.3 2-2.7 5.1-2.6 5.9-2.6h7.3c1 0 1.9-.3 1.9-1.5v-4.8c0-.8-.5-1.6-1.6-1.6h-8c-4 0-7.4 1.4-10 3.5-3.9 3.1-5.7 8.1-5.7 11.6 0 3.3.6 4.7 1.1 6.3.7 2.2 5.1 9.9 14.5 9.9h8.1c.7 0 1.8-.2 1.8-1.6-.3-1.7-.3-5.2-.3-5.2z" class="st2"/></svg>
```

Comment:
Please check the logo attached and update. Please move logo to solution folder, it should be under "Azure-Sentinel/Solutions/Contrast Security/Data Connectors/Logo" folder.